SMelidoni/react-app-collection
# feature/favicon-update

## Description of Changes

This PR updates the favicon icon used in the application.

## Type of Change


## Describe the problem this PR solves and your solution

The previous favicon icon was outdated and didn't align with the current branding guidelines. This PR replaces the old favicon icon with a new one that better represents the application and provides a refreshed visual identity.

## Testing Steps

- Open the application in a web browser.
- Verify that the favicon icon displayed in the browser tab has been updated to the new icon.
- Test the application's functionality to ensure that the favicon change does not impact the application's behavior.

## Screenshots

![joystick-32](https://github.com/SMelidoni/react-app-collection/assets/106830374/75860497-4d14-46a0-af61-5f207069dd4b)


## Checklist
